README.md

```diff
@@ -8,20 +8,24 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 ## Variables
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
-| aws_region | AWS region to use for resources. | string | - | yes |
-| cpu | The number of CPU units to reserve for the Shunting Yard container. Valid values can be 256, 512, 1024, 2048 and 4096. Reference: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `1024` | no |
-| memory | The amount of memory (in MiB) used to allocate for the Shunting Yard container. Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `4096` | no |
-| docker_image | Full path to Shunting Yard Docker image. | string | - | yes |
-| docker_version | Shunting Yard Docker image version. | string | - | yes |
-| allowed_s3_buckets | List of S3 Buckets to which Shunting Yard will have read-write access. eg. `["bucket-1", "bucket-2"]` | list | `<list>` | no |
-| instance_name | Shunting Yard instance name to identify resources in multi-instance deployments. | string | `` | no |
-| subnets | ECS container subnets. | list | - | yes |
+| allowed\_s3\_buckets | List of S3 Buckets to which SY will have read-write access. | list | n/a | yes |
+| aws\_region | AWS region to use for resources. | string | n/a | yes |
+| cpu | The number of CPU units to reserve for the Shunting Yard container. Valid values can be 256, 512, 1024, 2048 and 4096. Reference: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `"1024"` | no |
+| docker\_image | Full path of Shunting Yard Docker image. | string | n/a | yes |
+| docker\_version | Shunting Yard Docker image version. | string | n/a | yes |
+| graphite\_host | Graphite server configured in Shunting Yard to send metrics to. | string | `"localhost"` | no |
```

Comment:
I think the graphite options should go we should refer to the CT readme on what you can set in the config. Copying this here means we need to keep it in sync which hard, any change in CT on this might break it.